src/main/java/com/facebook/openwifirrm/RRMAlgorithm.java

```diff
@@ -98,7 +98,7 @@ public RRMAlgorithm(String name, Map<String, String> args) {
 	 *
 	 * @param name the algorithm name, which must exist in {@link AlgorithmType}
 	 * @param argsRaw the arguments as a comma-separated list of key=value
-	 *                pairs, e.g. <tt> key1=val1,key2=val2,key3=val3</tt>
+	 *                pairs, e.g. <code> key1=val1,key2=val2,key3=val3</code>
```

Comment:
Should I be worried about when to use `<code>` vs `{@code ...}`? I've always used `{@code ...}` since that's the only one I knew of to indicate that something is "source code"

Reply:
Oh you're right, I should have used `{@code ...}`, will update.